src/components/ToneBox/ToneBox.jsx

```diff
@@ -20,16 +20,16 @@ const styles = {
 const ToneBox = ({ toneResult }) => (
   <Card containerStyle={styles.container} className={classes.container}>
     <CardHeader style={styles.header} title="ALL SENTIMENTS" />
-    {toneResult.length ?
+    {toneResult ?
```

Comment:
Why did we take off length? Is this expected to be undefined? Might want to require at least an empty array to be passed in (can handle it one level up like `toneResult || []`